src/routes/onboarding/add-to-homescreen.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import smRectangle from "../../images/Smallrectangle.svg";
+import shareBar from "../../images/Sharebar.svg";
+import lgRectangle from "../../images/Largerectangle.svg";
+import icons from "../../images/Addtohomescreenicons.svg";
+
+export const AddToHomescreen: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-gradient-to-b from-[#F5F5F5] to-[#9ECCFA] overflow-auto max-w-screen-md pb-8 relative m-auto h-screen">
+      <h2 className="font-text text-section-header font-medium mt-5 leading-[52.8px] text-center max-w-[358px] mx-auto mb-8">
+        Add the app to your home screen to get push notifications
+      </h2>
+      <div className="flex items-center flex-col">
+        <div className="relative">
+          <img src={shareBar} alt="Share Button" />
+          <img src={smRectangle} alt="Click Share" className="absolute top-[1px] left-[163px]"/>

```

Comment:
I noticed the rectangle moves when the screen width drops below about 363px. this probably won't be an issue for *most* mobile screen resolutions, but I think the proper thing to do would be to correct this!

This might be hard to fix using CSS...considering the images also shrink when the screen edges start hugging. 
Since the rectangles are static, honestly the best option might be to just combine the rectangle image with the screenshot image into one.